opa/views/course_views.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="course_tree_view" model="ir.ui.view">
+        <field name="name">course.tree.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="title"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="course_search_view" model="ir.ui.view">
+        <field name="name">course.search.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="title"/>
+                <field name="description"/>
+                <filter name="my_courses" string="My Courses" domain="[('responsible_id', '=', uid)]"/>
+                <group string="Group By">
+                    <filter name="group_by_responsible" string="Responsible" context="{'group_by': 'responsible_id'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="course_form_view" model="ir.ui.view">
+        <field name="name">course.form.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form>
```

Comment:
Missing string

Reply:
@luisg123v string attribute on the views has not been deprecated?

Reply:
Comparing these two documentations:
https://vauxoo.github.io/odoo/reference/views.html

https://www.odoo.com/documentation/15.0/developer/reference/backend/views.html

Looks like in V15

View nodes that require string attribute:

- <activity>
- <cohort>
- <grid>

View nodes with optional use of string attribute

- <gantt>
- <graph>
- <pivot>

View nodes that don't use the string attribute:

- <calendar>
- <dashboard>
- <form>
- <kanban>
- <tree>
- <map>

@antonag32 please check that all your views to meet these criteria

cc: @luisg123v

Reply:
Thanks, I will check it.